ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -625,6 +625,40 @@ class SignalSpec: QuickSpec {
 				observer.sendNext("d")
 				expect(values) == [ "a", "cc", "d" ]
 			}
+
+			it("should not store strong reference to previously passed items") {
+				var disposedItems: [Bool] = []
+
+				struct Item {
```

Comment:
we may talk about strong refs in context of number of copies - structure hold strong ref to internal ScopedDisposable - and since someone holds copy of structure - this disposable will not be disposed.
We may change it to class and move this logic to deinit - but disposed items is only scope variable and it will be harder to handle insertion in deinit.